css/comments.css

```diff
@@ -0,0 +1,165 @@
+
+
+.typography ul.action-links, .typography ul.comments-list, .typography ul.no-bullet {
```

Comment:
I based the CSS on the default Simple theme and in some cases had to use .typography to override.  With modules what is the best practice regarding CSS?